pkg/apis/core/v1alpha1/types_schedulingprofile.go

```diff
@@ -26,7 +26,6 @@ import (
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:validation:Required
 // +kubebuilder:resource:path=schedulingprofiles,shortName=sp,singular=schedulingprofile,scope=Cluster
-// +kubebuilder:object:root=true
```

Comment:
Why did we remove this?

Reply:
Looks like `+k8s:deepcopy-gen` and `+k8s:deepcopy-gen:interface` have been deprecated in favor of `+kubebuilder:object:generate` and `+kubebuilder:object:root`.  https://book.kubebuilder.io/reference/markers/object.html

Reply:
it was a mistake

Reply:
fixed